packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -14,35 +14,46 @@
 #pragma push_macro("GetCurrentTime")
 #undef GetCurrentTime
 
+#include <DesktopWindowBridge.h>
 #include <winrt/Microsoft.ReactNative.h>
 
 #include <UI.Xaml.Controls.h>
 #include <UI.Xaml.Hosting.h>
 #include <winrt/Windows.Foundation.Collections.h>
-
 #pragma pop_macro("GetCurrentTime")
 
 #ifndef USE_WINUI3
 namespace xaml = winrt::Windows::UI::Xaml;
 #else
+#include <winrt/Microsoft.UI.h>
 namespace xaml = winrt::Microsoft::UI::Xaml;
 #endif
 
 namespace controls = xaml::Controls;
 namespace hosting = xaml::Hosting;
 
+#define WINDOWDATA L"WindowData"
```

Comment:
>#define WINDOWDATA  [](start = 0, length = 19)

We should avoid using macros for constants in modern C++. A better alternative is the constexpr or just const:
const wchar_t* WindowData = L"WindowData";

Reply:
Based on the code below we cannot use the `WindowData` as a property name. In that case we can name it `WindowDataProperty`.